common/service/config/config.go

```diff
@@ -419,6 +419,8 @@ type (
 
 	// BootstrapMode is an enum type for ringpop bootstrap mode
 	BootstrapMode int
+
+	RPCName string
```

Comment:
Generally we want to avoid making a type like this, it will make coding harder. We make types like this only when they are enums.